itou/templates/apply/process_refuse.html

```diff
@@ -1,72 +1,131 @@
 {% extends "apply/process_base.html" %}
 {% load django_bootstrap5 %}
+{% load theme_inclusion %}
 {% load buttons_form %}
 
-{% block content_extend %}
-    <div class="c-form">
-        <h2>Renseigner le motif de refus de candidature.</h2>
-        <form method="post" class="js-prevent-multiple-submit">
+{% block content_title %}
+    <div class="d-md-flex align-items-center mb-3">
+        <h1 class="mb-1 mb-md-0 me-3">
+            Refuser la candidature de <span class="text-muted">{{ job_application.job_seeker.get_full_name }}</span>
+        </h1>
+    </div>
+{% endblock %}
 
-            {% csrf_token %}
+{% block content %}
+    <section class="s-section">
+        <div class="s-section__container container">
+            <div class="row">
+                <div class="col-12 col-lg-8">
+                    <div class="c-stepper mb-3 mb-lg-5">
+                        <div class="progress progress--emploi">
+                            <div class="progress-bar progress-bar-{{ wizard|stepper_progress }}" role="progressbar" aria-valuenow="{{ wizard|stepper_progress }}" aria-valuemin="0" aria-valuemax="100">
+                            </div>
+                        </div>
+                        <p>
+                            {% if wizard.steps.current == "reason" %}
+                                <strong>Étape {{ wizard.steps.step1 }}</strong>/{{ wizard.steps.count }} : Choix du motif de refus
+                            {% elif wizard.steps.current == "job-seeker-answer" %}
+                                <strong>Étape {{ wizard.steps.step1 }}</strong>/{{ wizard.steps.count }} : Message au candidat
+                            {% elif wizard.steps.current == "prescriber-answer" %}
+                                <strong>Étape {{ wizard.steps.step1 }}</strong>/{{ wizard.steps.count }} : Message au prescripteur
+                            {% endif %}
+                        </p>
+                    </div>
+                    <div class="c-form mb-3 mb-lg-5">
+                        <form method="post" class="js-prevent-multiple-submit">
+                            {% csrf_token %}
+                            {{ wizard.management_form }}
 
-            {% bootstrap_form_errors form type="all" %}
+                            {% if wizard.steps.current == "reason" %}
+                                <h2 class="mb-3 mb-md-4">Choix du motif de refus</h2>
+                                <p class="mb-3 mb-md-4">
+                                    Dans le cadre d’un parcours IAE, la transparence sur les motifs de refus est importante pour le candidat comme le prescripteur. Nous vous encourageons à répondre à chacune des parties.
```

Comment:
Je trouve que 
```suggestion
                                    Dans le cadre d’un parcours IAE, la transparence sur les motifs de refus est importante pour le candidat comme pour le prescripteur. Nous vous encourageons à répondre à chacune des parties.
```
sonne mieux ?

Reply:
D'ailleurs vu qu'on a accès à `job_application.sender_kind`, on pourrait adapter le texte pour ne mentionner le prescripteur que si nécessaire ?